torch/csrc/distributed/rpc/init.cpp

```diff
@@ -115,10 +115,12 @@ Otherwise, throws an exception.
   // is cleaned up in join_rpc(), after join_rpc(), python objects returned
   // from rpc python call can not be resolved.
   auto futureMessage =
-      shared_ptr_class_<FutureMessage>(module, "FutureMessage")
+      shared_ptr_class_<torch::utils::Future<Message>>(module, "FutureMessage")
```

Comment:
Can we simply call this `Future` instead of `FutureMessage`? This way python users would just see a "Future" object and we don't leak "Message" to users.